inc/zoo/swar/associative_iteration.h

```diff
@@ -191,12 +191,17 @@ template<
     typename CountHalver
 >
 constexpr auto associativeOperatorIterated_regressive(
-    Base base, Base neutral, IterationCount count, IterationCount forSquaring,
-    Operator op, unsigned log2Count, CountHalver ch
+    const Base base,
+    const Base neutral,
+    IterationCount count,
+    const IterationCount forSquaring,
+    const Operator op,
+    unsigned log2Count,
+    const CountHalver ch
 ) {
     auto result = neutral;
-    if(!log2Count) { return result; }
-    for(;;) {
+    if (!log2Count) { return result; }
+    for (;;) {
```

Comment:
Please do not re-style code

Reply:
sorry, habit of fiddling while thinking.